### ISSUE TITLE
Minor change requests and a Big question. 

### ISSUE THREAD

**Issue by ahnHeejune** [shekkizh/FCN.tensorflow#88]
Thanks for sharing your implementation with the public.

I have couple of minor requests to update your code and one real question. 

1) how about using  'tqdm' when loading images and annotations or Chainer dataset.
2) the iteration is restarted from 0 when resuming the training. How about  using 'blobal-step' variable for fixing this.
3) also again, you 'd better use "os.pathsep" instead of '/' for file separators.

And my main question is.

I trained with the MIT dataset, but the training is not the same as you reported.
Train loss and Validation loss starts around 3.5 and does not decrease and just fluctuates.
In your report, it seems the loss should goes below 1.0 for convergence, am I right?
Step: 860, Train_loss:3.89231
Step: 870, Train_loss:3.64891
Step: 880, Train_loss:2.40985
Step: 890, Train_loss:3.11681
Step: 900, Train_loss:3.55415
Step: 910, Train_loss:3.38955
Step: 920, Train_loss:2.9828
Step: 930, Train_loss:2.39928
Step: 940, Train_loss:3.25557
Step: 950, Train_loss:3.52195
Step: 960, Train_loss:3.15188
Step: 970, Train_loss:5.03387
Step: 980, Train_loss:2.47206
Step: 990, Train_loss:3.91655
Step: 1000, Train_loss:2.78551
2018-03-24 15:37:13.298805 ---> Validation_loss: 3.48481

Looking forwards,

**Comment by suyanzhou626** [comment]
Do you solve it, My loss is like yours, if you sloved it, how to?

**Comment by EternityZY** [comment]
Me too. If you test this model by myself,the result is black photos.nothing information in this image.please help me!

**Comment by ahnHeejune** [comment]
I succeeded with other databases. So I think the implementation just does not work with the MIT dataset any more.

**Comment by suyanzhou626** [comment]
I finally succeed to train that FCN, but the result is a litte bad,  Maybe
the dataset don't suitable for FCN.

2018-04-25 19:28 GMT+08:00 Ahn, Heejune <notifications@github.com>:

**Comment by shekkizh** [comment]
Closing due to inactivity
